### PR TITLE
Checking for undefined textencoder

### PR DESCRIPTION
**Pull request by coder11**
Fixing `ReferenceError: TextEncoder is not defined` error

### CODE REVIEW

**Review by DieMyst — APPROVED** [review]

File: npm/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     },
     "devDependencies": {
         "@types/jest": "^26.0.23",
+        "@types/node": "^15.0.2",
```
Comment (folex) [inline npm/package.json:23]: Let's use 14 if possible

File: npm/src/wrapper.ts [hunk 2]
```diff
@@ -97,9 +99,9 @@ function getInt32Memory0(wasm) {
     return cachegetInt32Memory0;
 }
 
-let cachedTextDecoder = new TextDecoder('utf-8', { ignoreBOM: true, fatal: true });
+const lTextDecoder = typeof TextDecoder === 'undefined' ? module.require('util').TextDecoder : TextDecoder;
 
-cachedTextDecoder.decode();
```
Comment (folex) [inline npm/src/wrapper.ts:102]: is it OK that you don't call `decode` anymore?

Comment (coder11) [reply npm/src/wrapper.ts:102]: Well, seems like this line was added by accident

File: npm/package.json (hunk 1 above)
Comment (coder11) [reply npm/package.json]: ok

**Review by mikevoronov — APPROVED** [review]

**Review by folex — APPROVED** [review]

**Pull request merged**